packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -948,6 +970,44 @@ describe('Checkout', () => {
             });
         });
 
+        describe('loadBasket ::', () => {
+            describe('when `getBasket` request fails', () => {
+                let wrapper;
+
+                beforeEach(async () => {
```

Comment:
We don't really need the `beforeEach` in here and the case below, given there's only one test. I'd personally remove it, but up to you.

Reply:
Fixed